numpyro/contrib/nested_sampling.py

```diff
@@ -225,7 +224,7 @@ def run(self, rng_key, *args, **kwargs):
             for site in prototype_trace.values()
         )
         if has_enum:
-            from numpyro.contrib.funsor import enum, log_density as log_density_
+            from numpyro.contrib.funsor import enum
```

Comment:
could you revert the change here? this is required for enumeration.

Reply:
![Screenshot from 2023-03-07 19-28-30](https://user-images.githubusercontent.com/14807032/223595730-8aa54f17-e9f5-44e2-a190-d2554d9b25fa.png)
Ah I see, my code formatter removed it automatically. Okay, I won't run code formatting for IDE. Instead I'll use your `make format`.